nixos/lib/testing/nixos-test-base.nix

```
@@ -17,7 +17,10 @@ in
       # for external modules that test with e.g. testers.nixosTest and rely on that
       # version number.
       config.system.nixos.revision = mkForce "constant-nixos-revision";
-    }
 
+      # Avoid getting different hashes for test runs between classical nix and flake-enabled nix.
```

```suggestion
      # Avoid getting different hashes for test runs between the traditional entrypoints and flake entrypoints.
```

It's not a property of Nix. A flakes "enabled" Nix can evaluate expressions that use the traditional entrypoints, such as `eval-config.nix` or `nixos/lib`.